helm/csi-external-snapshotter-app/templates/defaults.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.defaults.volumesnapshotclass.create }}
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: {{ .Values.defaults.volumesnapshotclass.name }}
+  annotations:
+    snapshot.storage.kubernetes.io/is-default-class: "true"
+    helm.sh/hook: post-install, post-upgrade
```

Comment:
Why is that needed?

Reply:
Since we don't manage CRDs with `crds` folder, helm gives error in the first run for missing CRDs. helm doesn't validate the manifests after running pre-hooks.

Reply:
I mentioned this in the PR description.